### PR TITLE
Add a draft for the quarterly update

### PR DESCRIPTION
**Pull request by dnadales**

### CODE REVIEW

File: quarterly/2023-04-25-consensus.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#### Conway era
+
+- We integrated the Conway era into consensus.
+
+#### Technical debt
+
+- We fixed a bug with followers, which was discovered by property tests. 
+- We developed a DSL for specifying and running ChainDB test cases.
+- We fixed failing tests with iterators.
+- We created miro-benchmarks for adding transactions to the mempool.
```
Comment (Jasagredo) [inline quarterly/2023-04-25-consensus.md:53]: Typo: micro

File: quarterly/2023-04-25-consensus.md [hunk 2]
```diff
@@ -79,6 +79,22 @@ hide_table_of_contents: false
     - System-level tests not passing.
     - System-level benchmarks showing poor performance.
 
-#### Fostering collaboration 
+#### Support
+
+- Design Consensus side of hardfork-enactment in the Voltaire phase
+  ([#4180](https://github.com/input-output-hk/ouroboros-network/issues/4180)).
+- TODO Anything we need to plan for new crypto integration?
```
Comment (dnadales) [inline quarterly/2023-04-25-consensus.md:86]: @abailly

File: quarterly/2023-04-25-consensus.md [hunk 3]
```diff
@@ -79,6 +79,22 @@ hide_table_of_contents: false
...
+
+#### Tech debt
+
+- Check absence of thunks during nightly CI tests
+  ([#3001](https://github.com/input-output-hk/ouroboros-network/issues/3001) and
+  [#4006](https://github.com/input-output-hk/ouroboros-network/issues/4006))
+- Keep backwards compatibility with previous ledger snapshot versions
+  [#4434](https://github.com/input-output-hk/ouroboros-network/issues/4434).
+- Estimate the number of file descriptors Consensus needs
```
Comment (bartfrenk) [inline quarterly/2023-04-25-consensus.md:95]: This is currently filed under Support, see https://github.com/input-output-hk/ouroboros-consensus/issues/20.

Comment (dnadales) [reply quarterly/2023-04-25-consensus.md:95]: Done, thanks!

**Pull request merged**